.ci.prepare_codecov

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
CI fails because of this, does this file need x permissions or so?

Reply:
Oh maybe that, let me try.